### PR TITLE
Convert gotest to pytest

### PR DESCRIPTION
**Pull request by changli2023**

### CODE REVIEW

File: lightspark/__tests__/services/regtest/test_regtest_client_funding.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        LIGHTSPARK_RS_NODE_ID, seed_bytes, lightspark.BitcoinNetwork.REGTEST
+    )
+    payment = client.create_test_mode_payment(
+        LIGHTSPARK_RS_NODE_ID, invoice.data.encoded_payment_request
+    )
+    log.info("Created payment: %s", payment)
+    tx = wait_for_payment_completion(client, payment.id)
+    assert tx.status != lightspark.TransactionStatus.FAILED, "Payment failed!"
+    log.info(tx)
+    log.info("===test_funding_node1===SUCCEEDED\n")
```
Comment (waterson) [inline lightspark/__tests__/services/regtest/test_regtest_client_funding.py:32]: Does this actually print out anything when running as a pytest? (Do we need it to? Isn't the fact that the test passed sufficient?)

File: lightspark/__tests__/test_config.py
Comment (waterson) [inline lightspark/__tests__/test_config.py:1]: Do we actually want to check this in to the SDK? Isn't this going to be public? Not that the keys are super secret or anything, but... seems weird that we'd put this into the SDK. For example, if we have to recreate the nodes / change keys we'll need to do an SDK release, right?

**Review by waterson — COMMENTED** [review]
I think it's good that we have these tests in the Python SDK, because -- sure, we'll want to test the Python SDK.

But won't we eventuallyl need some internal version of this that allows us to use the sparkcore APIs for more elaborate testing?

**Comment by vdurmont** [comment]
[runningaway](https://app.graphite.dev/macros/lightsparkdev)

**Pull request closed without merging**